orderbook/src/api/get_orders.rs

```diff
@@ -29,6 +29,8 @@ struct Query {
     include_insufficient_balance: bool,
     #[serde(default)]
     include_unsupported_tokens: bool,
+    #[serde(default)]
+    include_signature_pending: bool,
```

Comment:
Nit: this parameter is only meaningful for presignatures, so I'd prefer this to be reflected in the name: `include_presignature_pending`. Especially since *pre*signature is used in a similar variable somewhere else in the PR.

Reply:
The naming discrepancy was international as I figured this may be used for EIP-1271 as well. That being said, now that I think about it more - we should probably use `include_presignature_pending` for now, and rename later if we end up needing something like this for EIP-1271 as well.